docs/src/ToricVarieties/ToricDivisorClasses.md

```diff
@@ -37,18 +37,24 @@ Base.:*(c::fmpz, td::ToricDivisorClass)
 Base.:(==)(tdc1::ToricDivisorClass, tdc2::ToricDivisorClass)
 ```
 
-
 ## Properties of toric divisor classes
 
 ```@docs
 istrivial(tdc::ToricDivisorClass)
 ```
 
-
 ## Operations for toric divisor classes
 
 ```@docs
 divisor_class(tdc::ToricDivisorClass)
 toric_variety(tdc::ToricDivisorClass)
 toric_divisor(tdc::ToricDivisorClass)
 ```
+
+## Special divisor classes
+
+```@docs
+TrivialDivisorClass(v::AbstractNormalToricVariety)
+AnticanonicalDivisorClass(v::AbstractNormalToricVariety)
+CanonicalDivisorClass(v::AbstractNormalToricVariety)
```

Comment:
Probably these should be `snake_case` too. Actually at this point I wonder about things like `NormalToricVariety`, `PolyhedralFan`, etc.

Maybe @thofma or @fingolfin can give some input and I'd try to update the styleguide accordingly?

Reply:
Yes, they should be all `snake_case`. Not sure why they should be `CamelCase` in the first place.

Reply:
Nothing is ambiguous, it is just about the distinction between constructors and functions.

Reply:
Ok. `snake_case` it is.